projects/stretch_ovmm/eval_episode.py

```diff
@@ -52,6 +49,10 @@ def main(
     print("- Loading configuration")
     config = load_config(visualize=visualize_maps, **kwargs)
 
+    REAL_WORLD_CATEGORIES[1] = pick_object
+    REAL_WORLD_CATEGORIES[2] = start_recep
+    REAL_WORLD_CATEGORIES[3] = goal_recep
+
```

Comment:
```suggestion
    REAL_WORLD_CATEGORIES[1] = pick_object
    REAL_WORLD_CATEGORIES[2] = start_recep
    REAL_WORLD_CATEGORIES[3] = goal_recep

```

Reply:
We can remove this - I think this was deleted in OVMM int?